### DISCUSSION TITLE
mill-build.com down? mill-build.org still up.

### DISCUSSION THREAD

**Discussion by roman-mibex-2** [com-lihaoyi/mill#3343] [Development]
Looks like mill-build.com is down. However, mill-build.org still does work.

**Comment by roman-mibex-2** [comment]
AHA. The issue is that:

- http://mill-build.com -> redirects to mill-build.org and is working
- https://mill-build.com -> Does not answer

My browser defaults to https, so the site looks 'dead'.

**Comment by roman-mibex-2** [comment]
Website got moved, see:https://github.com/com-lihaoyi/mill/discussions/3339

**Comment by lihaoyi** [comment]
Ah the namecheap redirect doesnt work for https. Looks like I'll  have to manually set up some redirect pages
